official/resnet/resnet_run_loop.py

```diff
@@ -384,16 +405,37 @@ def input_fn_eval():
       benchmark_logger = logger.BenchmarkLogger(flags.benchmark_log_dir)
       benchmark_logger.log_estimator_evaluation_result(eval_results)
 
+  # Return the classifier after all epochs are completed.
+  return classifier
+
+
+def export_savedmodel(classifier, export_dir, shape, batch_size):
+  """Exports a saved model for the given classifier."""
+  input_receiver_fn = export.build_tensor_serving_input_receiver_fn(
+      shape, batch_size=batch_size)
+  classifier.export_savedmodel(export_dir, input_receiver_fn)
+  return classifier
```

Comment:
Is there any reason to return the classifier here? Since the export_savedmodel is not called on classifier instance, I don't think it can help chaining the calls.

Reply:
Consistency with the other function is what I was thinking, but I'm not attached to it. Will remove.